File: rock63/src/main/java/com/uksusoff/rock63/data/UserPrefs.java
package com.uksusoff.rock63.data;

import org.androidannotations.annotations.sharedpreferences.DefaultBoolean;
import org.androidannotations.annotations.sharedpreferences.SharedPref;

@SharedPref
public interface UserPrefs {

    @DefaultBoolean(true)
    boolean remindWeekBefore();

    @DefaultBoolean(true)
    boolean remindDayBefore();

}

File: rock63/src/main/java/com/uksusoff/rock63/data/UserPrefs.kt
package com.uksusoff.rock63.data

import org.androidannotations.annotations.sharedpreferences.DefaultBoolean
import org.androidannotations.annotations.sharedpreferences.SharedPref

/**
 * Created by Vyacheslav Vodyanov on 18.06.2016.
 */
@SharedPref
interface UserPrefs {
    @DefaultBoolean(true)
    fun remindWeekBefore(): Boolean

    @DefaultBoolean(true)
    fun remindDayBefore(): Boolean
}